packages/modules/src/document/Document.ts

```diff
@@ -371,7 +371,7 @@ export async function fromContent({
   return document
 }
 
-export async function updateStream(
+export async function updateFromContent(
```

Comment:
Any reason we are not following the same function style as fromContent here? Also, the update will be either at the content level or the options - like the expiry date change. If the content is not changed, we don't need to process the content. Also, if the content is changed, we should validate it against the schema mentioned in the original document.